examples/model_compress/pruning/amc/amc_train.py

```diff
@@ -23,8 +23,13 @@
 from utils import AverageMeter, accuracy, progress_bar
 
 sys.path.append('../../models')
-from mobilenet import MobileNet
-from mobilenet_v2 import MobileNetV2
+try:
+    from mobilenet import MobileNet
+    from mobilenet_v2 import MobileNetV2
+except ModuleNotFoundError as e:
+    print(e)
+    print('Please check whether the current working path is the path where this file is located.')
```

Comment:
Then I would suggest `sys.path.append(Path(__file__).parents[2]`.

Reply:
Good suggestion, fix it